app/src/main/java/com/kickstarter/ui/activities/HelpSettingsActivity.kt

```diff
@@ -85,4 +74,18 @@ class HelpSettingsActivity : BaseActivity<HelpSettingsViewModel.ViewModel>() {
             startActivity(Intent.createChooser(intent, getString(R.string.support_email_chooser)))
         }
     }
+
+    private fun loadChromePages(endpoint: String) {
+        val webEndpointBuilder = Uri.parse(this.environment().webEndpoint()).buildUpon()
+        webEndpointBuilder.appendEncodedPath(endpoint)
+
+        val url = webEndpointBuilder.build().toString()
+        val builder = CustomTabsIntent.Builder()
+
+        builder.setShowTitle(true)
+        builder.setToolbarColor(ContextCompat.getColor(this, R.color.primary))
+
+        val customTabsIntent = builder.build()
+        customTabsIntent.launchUrl(this, Uri.parse(url))
```

Comment:
This crashes if the user doesn't have a browser that's capable of handling a custom tabs.  I don't think we needed to remove all of that code. We need a fallback.